gitlab/const.py

```diff
@@ -27,17 +30,48 @@
 MAINTAINER_ACCESS: int = 40
 OWNER_ACCESS: int = 50
 
+
```

Comment:
I think all of these new `Enum` (hopefully `IntEnum`) classes should be at the top of the file. And then update the module level constants to get their values from the new `Enum` class.

For example:

```
NO_ACCESS = AccessLevel.NO_ACCESS.value
```

Maybe better to do it in two commits, but in the same PR.

What do you think?

Reply:
yes, done.